packages/forms/resources/js/components/select.js

```diff
@@ -112,16 +113,16 @@ export default (Alpine) => {
                             async (event) => {
                                 let search = event.detail.value?.trim()
 
-                                if ([null, undefined, ''].includes(search)) {
-                                    return
-                                }
+                                let label = [null, undefined, ''].includes(search)
```

Comment:
Label can be const. Actually there's no reason to assign it to a variable first, is there?